tox.ini

```diff
@@ -68,6 +74,8 @@ commands =
     {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_2} --pattern "{[local_settings]test_pattern}"
     {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_3} --pattern "{[local_settings]test_pattern}"
     {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_4} --pattern "{[local_settings]test_pattern}"
```

Comment:
I think you need to add the fixture paths to the first few commands instead of below them, e.g. see the order at https://github.com/rcsb/py-rcsb_exdb/blob/master/tox.ini#L77-L80